packages/CopyInput/src/CopyInput.js

```diff
@@ -0,0 +1,69 @@
+import React from "react";
+import PropTypes from "prop-types";
+import Input from "@paprika/input";
+import Popover from "@paprika/popover";
+import Button from "@paprika/button";
+import useI18n from "@paprika/l10n/lib/useI18n";
+import extractChildrenProps from "@paprika/helpers/lib/extractChildrenProps";
+import CopyIcon from "@paprika/icon/lib/ArrowDown"; // TODO: Replace with actual icon when ready
+import CopyInputInputPropsCollector from "./CopyInputInputPropsCollector";
+import * as sc from "./CopyInput.styles";
+
+function CopyInput(props) {
+  const { defaultValue, children, ...moreProps } = props;
+  const I18n = useI18n();
+  const extendedInputProps = extractChildrenProps(children, CopyInputInputPropsCollector);
+
+  const [value, setValue] = React.useState(defaultValue);
+  const [isClickedTooltipOpen, setIsClickedTooltipOpen] = React.useState(false);
+
+  function handleButtonClick() {
+    navigator.clipboard.writeText(value);
```

Comment:
can you add a note on the ReadMe file that this doesn't support `ie11`? https://caniuse.com/mdn-api_navigator_clipboard 

This might be fine for now since this is primarily for next-gen AN which has end support for ie11 but if this would be used in results or another part of our platform this might not be enough. 

you would need to do something like this to make it work in IE11 as well:

```js
function copyToClipboard(element) {
    var $temp = $("<input>");
    $("body").append($temp);
    $temp.val($(element).text()).select();
    document.execCommand("copy");
    $temp.remove();
}
```

Reply:
We're going to use this component in Robots as well, but I'm not sure if Robots will need to support ie11 either since it's going to be pretty tied with the flow builder.

Reply:
Ohh I see, so maybe two options:
1. Testing in ie11 is not that simple with our current gear because the storybook doesn't render in ie11 :/, so a possibility is let it like this add a note in the readme pointing to this PR and if we later need to support it, we can let an ux-dev add the support and properly test it in ie11.

2. We replaced your current code with a similar that I'm suggesting and then we just asked a ux-dev later to test it in ie11 whenever they need to implemented in that env to check if it's working correctly.

What do you think?

Reply:
I think it makes the most sense to use code compatible with ie11 now.  Some customers are going to continue to use ie11, and will suffer degraded capabilities in some places, but if this works with vanilla js, why not?

I did implement a vanilla js version, with in important a11y detail (can you see it?) that you may want to copy over:
https://github.com/acl-services/paprika/blob/002d64e1d4dea17e1530c3ba2c25a3279cc47645/.storybook/components/CodeViewer/CodeViewer.helpers.js#L31

Reply:
sound good if @robyn3choi can add that would be 💯 🙇